react/src/components/atoms/media/SiteLogo/index.js

```diff
@@ -12,7 +12,7 @@ const SiteLogo = (siteLogo) => (
   <div className="ma__site-logo">
     <a href={siteLogo.url.domain ? siteLogo.url.domain : '/'} title={siteLogo.title}>
```

Comment:
```suggestion
    <a href={siteLogo?.url?.domain ? siteLogo.url.domain : '/'} title={siteLogo.title}>
```
should this be using the same syntax for conditional chaining?